### DISCUSSION TITLE
After I copied the runas.exe to WSL, it doesn't work as expected, why?

### DISCUSSION THREAD

**Discussion by wzis** [community/community#20308] [Discussions]
> See following, in WSL2 env, I first ran runas.exe without using the copied version, and it works:
> gwz@AMD-PC:~/myapp/whc$ runas.exe /user:wzis notepad.exe
> Enter the password for wzis:
> Attempting to start notepad.exe as user "AMD-PC\wzis" ...
> 
> Then I tried the copied version, it did ask for the password of the target user, but after that, it didn't try to start notepad.exe as the target user:
> gwz@AMD-PC:~/myapp/whc$ ./runas.exe /user:wzis notepad.exe
> Enter the password for wzis:
> gwz@AMD-PC:~/myapp/whc$
> 
> Does anyone know why?
> Thanks in advance.

**Comment by FahdSeddik** [comment]
Hey!

I hope this [link](https://social.technet.microsoft.com/Forums/en-US/1d8b248e-0ac5-463a-97a9-8015bb66c9ed/runasexe-does-not-work-properly-with-paste-from-clipboard-as-of-windows-10-v1703?forum=win10itprogeneral) contains useful information for you!

**Reply by wzis** [reply]
Thanks for the info. I checked it's a bit different, in that one, it's the password not accepted, but in my one, it seems it did accept the password I entered but after that nothing happened, just exited.
